slice/slice.go

```diff
@@ -0,0 +1,13 @@
+package slice
+
+func UniqueSlice[S ~[]E, E comparable](s S) S {
```

Comment:
```suggestion
func Unique[S ~[]E, E comparable](s S) S {
```

Its a little bit verbose considering the package name is already slice.

Reply:
done